packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.native.test.ts

```diff
@@ -0,0 +1,67 @@
+import { getBaseBoxStyles, getResponsiveValue, getSpacingValue } from './getBaseBoxStyles';
+import type { BaseBoxProps } from './types';
+import paymentLightTheme from '~components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme';
+
+describe('getResponsiveValue', () => {
+  it('should return correctly for plain values', () => {
+    expect(getResponsiveValue('hello', 'base')).toBe('hello');
```

Comment:
Ok so this function has a bit different behaviour on web vs native (it is expect because on web we want media-query friendly values and on react native, we just want to return one value always depending on priority we have defined for mobile screen size)

On web, it returns the value for given size like this-
```ts
getResponsiveValue('hello', 'base') // -> 'hello'
getResponsiveValue({ base: 3, s: 0 }, 'base') // -> 3
getResponsiveValue({ base: 3, s: 0 }, 's') // -> 0
```

This helps on web because I can now call this with different screen sizes to create media query css-
```js
const props = {
  display: 'block';
}

const pseudoCSS = {
  display: getResponsiveValue(props.display, 'base');
  '@media (min-width: s)': {
      display: getResponsiveValue(props.display, 's');
  },
  '@media (min-width: m)': {
      display: getResponsiveValue(props.display, 'm');
  },
}
```

But on native, we just have this without defined size (size has default value as `base`) and then it returns the value depending on that priority we defined 
```js
const pseudoStylesObject = {
  display: getResponsiveValue(props.display);
}
```

In this particular example, I am just passing size anyway. We can choose to not pass.

Reply:
Can we change the function signature to accept an object like `getResponsiveValue({ value, size })`

Its not too obvious on what this function does unless I dig into it

Reply:
Also is this exposed to the consumers?

Reply:
Can you check this comment once https://github.com/razorpay/blade/pull/1009#issuecomment-1447696941 and see if it looks convincing? if everyone else feels it's unreadable I can change it to object.

Also can you check this in usage instead of tests? In tests I have used hardcoded strings and random values so it might be getting confusing here.

This is how it looks in usage
<img width="776" alt="image" src="https://user-images.githubusercontent.com/30949385/223021889-707f5966-8b9d-4acd-b6d1-8731bb3b2318.png">


> Also is this exposed to the consumers?

No this isn't. Its only limited to `getBaseBoxStyles` file